src/components/media-views.js

```diff
@@ -706,6 +706,8 @@ AFRAME.registerComponent("media-image", {
 
   async update(oldData) {
     let texture;
+    let width = 1;
```

Comment:
Is it a bug if this default value is used? If so, would prefer that they are not given any default width/height so that we can notice that bug if it happens.

Reply:
Ah I thought the error image was square, but actually its not, so it should default to the size of that.